### ISSUE TITLE
Cannot read properties of undefined (reading 'autoFocus'). Not able to find the solution for it. Please help. 

### ISSUE THREAD

**Issue by Yashu1205** [hypeserver/react-date-range#535]
## Subject of the issue
Describe your issue here.
 
## [BUG] Bug Reproduce Steps
Tell us how to reproduce this issue.

## Environment
Package Version:
React version:
Node version:
Browser:

**Comment by malkeshfyn** [comment]
did you get a solution?

**Comment by Yashu1205** [comment]
Yes, I did. I actually don't remember the solution but I think it was an
issue in the local implementation itself.

Thanks

**Comment by malkeshfyn** [comment]
thanks ,if possible share the solution
